subsys/bluetooth/shell/audio.c

```diff
@@ -1759,25 +1759,36 @@ static int cmd_context(const struct shell *sh, size_t argc, char *argv[])
 	enum bt_audio_dir dir;
 	enum bt_audio_context ctx;
 
-	if (!strcmp(argv[1], "sink")) {
+	if (!strcmp(argv[2], "sink")) {
```

Comment:
This does not match the help below:
```
	SHELL_COND_CMD_ARG(CONFIG_BT_PACS, set_context, NULL,
			   "<direction: sink, source>"
			   " <type: supported, available> <context bitmask>",
			   cmd_context, 4, 0),
```

According to which, the direction `sink, source` is 2nd argument (`argv[1]`), while `type` is 3rd (`argv[2]`).
Besides that, please parse the positional arguments in order.

Reply:
Done